accumulate/full/enqueue_and_unlock_chain-3.json

```diff
@@ -0,0 +1,2860 @@
+{
+    "input": {
+        "slot": 46,
+        "reports": [
```

Comment:
reports in input have the same core_index = 0. Is this possible? Shouldn't exist only one report per core?

Reply:
Good catch! These were manually generated, so some errors might have slipped through. I'll fix them as soon as I'm back.

Reply:
@danicuki  Can you give these another shot? Ty

Reply:
it is working fine here! Thanks!